app/assets/javascripts/shared/datatable/state.js

```diff
@@ -1,59 +1,83 @@
 $(document).on('preInit.dt', 'body.issues.index', function (e, settings) {
   var $paths = $('[data-behavior~=datatable-paths]');
-  if ($paths.data('table-state-url') == undefined) { return; }
+  if ($paths.data('table-state-url') == undefined) {
+    return;
+  }
+
+  var api = new $.fn.dataTable.Api(settings);
 
-  var api = new $.fn.dataTable.Api( settings );
   api.button().add(1, {
     attr: {
-      'data-behavior': 'table-action'
+      'data-behavior': 'table-action',
     },
     autoClose: true,
     className: 'd-none',
     extend: 'collection',
     name: 'stateBtn',
     text: '<i class="fa fa-adjust fa-fw"></i>State<i class="fa fa-caret-down fa-fw"></i>',
-    buttons: DradisDatatable.prototype.setupStateButtons.call(api)
+    buttons: DradisDatatable.prototype.setupStateButtons.call(api),
   });
 });
 
 $(document).on('init.dt', 'body.issues.index', function (e, settings) {
   var $paths = $('[data-behavior~=datatable-paths]');
-  if ($paths.data('table-state-url') == undefined) { return; }
+  if ($paths.data('table-state-url') == undefined) {
+    return;
+  }
 
-  var api = new $.fn.dataTable.Api( settings );
-  api.on('select.dt deselect.dt', function() {
-    var selectedCount = this.rows({selected:true}).count();
-    DradisDatatable.prototype.toggleStateBtn.call(api, selectedCount !== 0);
-  }.bind(api));
+  var api = new $.fn.dataTable.Api(settings);
+  api.on(
+    'select.dt deselect.dt',
+    function () {
+      var selectedCount = this.rows({ selected: true }).count();
+      DradisDatatable.prototype.toggleStateBtn.call(api, selectedCount !== 0);
+    }.bind(api)
+  );
 });
 
-DradisDatatable.prototype.setupStateButtons = function() {
-  var states = [['Draft', 'fa-pencil-square-o'], ['Published', 'fa-rocket']],
+DradisDatatable.prototype.setupStateButtons = function () {
+  var states = [
+      ['Draft', 'fa-pencil-square-o'],
+      ['Ready for review', 'fa-eye'],
+      ['Published', 'fa-rocket'],
```

Comment:
Shouldn't we bake this as data- attributes from the model to avoid having to chase it if the collection changes in the future?

Reply:
Makes sense to keep it all together in the model, I have pushed this change.